plugins/webhooks/index.js

```diff
@@ -78,7 +78,8 @@ function startPRJob(options, request) {
                 type: 'pr',
                 workflow: [job.name],
                 username,
-                sha
+                sha,
+                causeMessage: `${options.action} by ${username}`
```

Comment:
We would have to click the PR-## link to get to the PR build page, so I don't think it's necessary to add `Pull request` to the cause message